README.md

```diff
@@ -54,6 +54,11 @@ sidekiq-monitoring for email-alert-api uses 3089
 * Run `RAILS_ENV=test bundle exec rake db:setup` to load the database
 * Run `bundle exec rspec` to run the tests
 
+### Using test email addresses for signup
+
+Using any email address that ends with '@notifications.service.gov.uk'
+will not create a subscriber or a subscription, however will return a 201.
+
```

Comment:
This sounds accurate but I think it would make more sense being in the context of the application endpoints section as this only applies to that particular endpoint.